app/monitoringapi.go

```diff
@@ -42,8 +42,8 @@ var (
 	errReadyInsufficientPeers = errors.New("quorum peers not connected")
 	errReadyBeaconNodeSyncing = errors.New("beacon node not synced")
 	errReadyBeaconNodeDown    = errors.New("beacon node down")
-	errReadyVCNotConfigured   = errors.New("vc not configured")
-	errReadyVCMissingVals     = errors.New("vc missing some validators")
+	// errReadyVCNotConfigured   = errors.New("vc not configured")
+	// errReadyVCMissingVals     = errors.New("vc missing some validators").
```

Comment:
golangci-lint being proactive not helping here 😂
```suggestion
	// errReadyVCMissingVals     = errors.New("vc missing some validators")
```